src/objects/grid_obj.f90

```diff
@@ -244,10 +245,10 @@ subroutine update_with_halos(grid, halo_size)
       grid%jme = grid%jme + merge(0, halo_size, north_boundary)
 
       ! if this is on a boundary, we should skip 1 grid cell (the boundary conditions) else we should skip the halo
-      grid%its = grid%ims + merge(1, halo_size, west_boundary)
-      grid%ite = grid%ime - merge(1, halo_size, east_boundary)
-      grid%jts = grid%jms + merge(1, halo_size, south_boundary)
-      grid%jte = grid%jme - merge(1, halo_size, north_boundary)
+      grid%its = grid%ims + halo_size !merge(1, halo_size, west_boundary)
```

Comment:
are boundary's now handled elsewhere?

Reply:
or do you always need a halo width on all sides regardless of boundary location now?